db/structure.sql

```diff
@@ -70,13 +51,11 @@ CREATE EXTENSION IF NOT EXISTS "uuid-ossp" WITH SCHEMA public;
 COMMENT ON EXTENSION "uuid-ossp" IS 'generate universally unique identifiers (UUIDs)';
 
 
-SET search_path = public, pg_catalog;
-
 --
 -- Name: update_updated_at_column(); Type: FUNCTION; Schema: public; Owner: -
 --
 
-CREATE FUNCTION update_updated_at_column() RETURNS trigger
+CREATE FUNCTION public.update_updated_at_column() RETURNS trigger
```

Comment:
I'm not really sure why these are now marked as public. I assume it's something to do with either the ActiveRecord upgrade or the Postgres upgrade.